R/config.R

```diff
@@ -88,23 +87,9 @@ validate_config_vs_hub_tasks <- function(hub_path, webevals_config) {
   }
 
   task_groups <- hub_tasks_config[["rounds"]][[1]][["model_tasks"]]
-  target_ids_by_task_group <- purrr::map(
```

Comment:
Moved this out into its own small function in `hub_tasks_config_utils.R`, and called it from within `validate_config_targets`, which is the only place in `config.R` where it was needed.  The function is also called from the new `load_model_out_in_window.R` below.